examples/dswx_s1_sample_runconfig-v3.0.0-rc.1.0.yaml

```diff
@@ -214,4 +214,19 @@ RunConfig:
             # DSWx-S1 product format
             output_imagery_format: 'COG'
 
-          log_file:
+          browse_image_group:
+            # Save a full-res Cloud-Optimized GEOTIFF DSWx-S1 browse image and
+            # a modified-resolution PNG of the browse image for DSWx-S1
+            save_browse: True
+
+            # Setting `browse_image_height` and `browse_image_width` equal
+            # will maintain this original DSWx-S1 aspect ratio
+            # of 3660 pixels x 3660 pixels for the PNG browse image.
+            # If these fields are left empty, 1024 x 1024 will be used.
+            # Height in pixels for the PNG browse image
+            browse_image_height: 1024
+
+            # Width in pixels for the PNG browse image
+            browse_image_width: 1024
+
```

Comment:
Just to be clear and to make sure I understand the comment.  Are we setting the default referred to by "if these fields are left empty....", and the user would be responsible if they wanted higher resolution?

Reply:
Since this field is in the SAS portion of the RunConfig, the SAS would be responsible for setting the default value if left blank.